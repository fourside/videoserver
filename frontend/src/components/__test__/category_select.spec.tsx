import * as React from 'react';
import 'jest';
import * as renderer from 'react-test-renderer';

import CategorySelect from '../category_select';

test('snapshot', () => {
  const props = {
    current: "",
    categories: [""],
    onChange: () => {},
  };
  const component = renderer.create(<CategorySelect {...props} />).toJSON();
  expect(component).toMatchSnapshot();
});
