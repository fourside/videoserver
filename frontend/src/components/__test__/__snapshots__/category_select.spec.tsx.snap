// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="control has-icons-left"
>
  <div
    className="select is-small"
  >
    <select
      onChange={[Function]}
      value=""
    >
      <option
        value=""
      >
        all
      </option>
      <option
        value=""
      >
        
      </option>
    </select>
  </div>
  <div
    className="icon is-small is-left"
  >
    <i
      className="fas fa-tag"
    />
  </div>
</div>
`;
